pages/api/reservoir/[...slug].ts

```diff
@@ -33,6 +34,25 @@ const proxy = async (req: NextApiRequest, res: NextApiResponse) => {
   setParams(url, query)
 
   if (endpoint.includes('redirect/')) {
+    // Redirect eth and weth currency icons to self-hosted
+    // versions without any padding
+    endpoint = endpoint.toLowerCase()
+
+    if (
+      (chainPrefix == 'ethereum' || chainPrefix == 'goerli') &&
+      endpoint.includes('currency')
+    ) {
+      if (endpoint.includes(constants.AddressZero)) {
+        res.redirect('/icons/currency/new-eth.png')
```

Comment:
Looks good, maybe we should change the name from new-eth and new-weth to something more descriptive like no-padding-eth etc.